pom.xml

```diff
@@ -128,12 +128,33 @@
   <build>
     <plugins>
       <plugin>
+        <groupId>org.codehaus.mojo</groupId>
```

Comment:
I'm not sure why the compiler plugin doesn't fail. I tested it myself right now, adding the `tzAt` method back and it compiles successfully, even though the target JDK is 1.7.